l10n/css.json

```diff
@@ -384,9 +384,8 @@
     "fr": "telle que spécifiée, sauf avec &lt;resolution&gt; éventuellement modifiée lors du calcul par la valeur 'snap'",
     "ru": "как указано, за исключением &lt;resolution&gt; может изменить значением 'snap'"
   },
-  "asSpecifiedWithLengthsMadeAbsolute": {
-    "de": "wie angegeben, wobei Längen in absolute Werte umgewandelt werden",
-    "en-US": "as specified with lengths made absolute"
+  "asSpecifiedWithLengthsAbsoluteAndNormalComputingToZeroExceptMultiColumn": {
+    "en-US": "as specified, with <length>s made absolute, and normal computing to zero except on multi-column elements"
```

Comment:
I think this should encode `<` and `>` in `<length>`, shouldn't it?

Reply:
You mean like `&gt;` type of thing?

Reply:
Yes, exactly. We do this in other places, e.g. https://github.com/mdn/data/blob/master/l10n/css.json#L163, and we also include markup in these strings, e.g. https://github.com/mdn/data/blob/master/l10n/css.json#L176, so it seems like we're just inserting these strings into the HTML output for MDN.